pkg/framework/framework.go

```diff
@@ -13,3 +13,7 @@ func HASSuiteDescribe(text string, body func()) bool {
 func CommonSuiteDescribe(text string, body func()) bool {
 	return Describe("[common-suite "+text+"]", Ordered, body)
 }
+
+func ChainsSuiteDescribe(text string, body func()) bool {
+	return Describe("[chains-suite "+text+"]", Ordered, body)
+}
```

Comment:
Since this is same as `CommonSuiteDescribe` and the `text` below is provided with `"Tekton Chains E2E tests"` perhaps we don't need, i.e. can use `CommonSuiteDescribe` below instead?

Reply:
There is a `HASSuiteDescribe` which is also very similar to `CommonSuiteDescribe`. The value of these functions seem to be to add the corresponding prefix. Maybe it's best to stick with the existing pattern, but provide a less redundant `text` when calling this function?